src/select-field.jsx

```diff
@@ -141,6 +141,8 @@ const SelectField = React.createClass({
       hintText,
       fullWidth,
       errorText,
+      onFocus,
```

Comment:
@oliviertassinari @cgwyllie `onFocus` and `onBlur` are not officially exposed as events for the dropdown menu component ([doc page](http://material-ui.com/#/components/dropdown-menu)). Plus I agree that attaching those handlers to a `div` is rather useless.

So I would say feel free to remove it. Anyway, our next release is v0.14.0 -- so minor breaking changes are okay.